frontend/components/product-list/ProductListView.tsx

```diff
@@ -39,7 +39,9 @@ export function ProductListView({
                   px={{ base: 3, sm: 0 }}
                >
                   <ProductListBreadcrumb productList={productList} />
-                  <ProductListDeviceNavigation productList={productList} />
+                  {productList.title !== 'Repair Tools' && (
```

Comment:
This works but it's a little bit brittle since if the marketing team decides to update the title this check would need to be updated too.
https://github.com/iFixit/react-commerce/pull/380 introduces a `type` field on the product list that you can use to check if the collection is the tools root, so you may want to wait for that PR to be merged and update the condition accordingly